packages/ui-navigation/package.json

```diff
@@ -46,6 +46,8 @@
     "@instructure/ui-tooltip": "^7.3.1",
     "@instructure/ui-utils": "^7.3.1",
     "@instructure/ui-view": "^7.3.1",
+    "@instructure/emotion": "^7.3.1",
+    "@instructure/ui-themes": "^7.3.1",
     "classnames": "^2",
```

Comment:
Unnecessary classnames

Reply:
It is needed until the navigation component is not migrated